source/hwIo/ioThread.py

```diff
@@ -197,7 +197,10 @@ def fakeApc(param):
 		self.handle = None
 
 	def run(self):
-		while True:
-			ctypes.windll.kernel32.SleepEx(winKernel.INFINITE, True)
-			if self.exit:
-				break
+		try:
+			while True:
+				ctypes.windll.kernel32.SleepEx(winKernel.INFINITE, True)
+				if self.exit:
+					break
+		except Exception:
+			log.critical("Exception in IoThread function", exc_info=True, stack_info=True)
```

Comment:
Using `exc_info=True` and  `stack_info=True)` seems unusual. What is expected?

Reply:
Well, it should log the exception as well as a strack trace.

Reply:
Actually, good catch. I want to log not only a stack trace from the IoThread, but dump the stack of all threads, like the watchdog does.